hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconServerConfigKeys.java

```diff
@@ -143,6 +143,19 @@ public final class  ReconServerConfigKeys {
       "ozone.recon.scm.connection.request.timeout";
   public static final String
       OZONE_RECON_SCM_CONNECTION_REQUEST_TIMEOUT_DEFAULT = "5s";
+
+  public static final String OZONE_RECON_SCM_SNAPSHOT_TASK_INTERVAL_DELAY =
+      "ozone.recon.scm.snapshot.task.interval.delay";
+
+  public static final String OZONE_RECON_SCM_SNAPSHOT_TASK_INTERVAL_DEFAULT
+      = "10m";
```

Comment:
same as ozone xml configuration, this periodic sync can be with lesser frequency as once a day

Reply:
Changes done to 24h @sumitagrawl